Jenkinsfile_CNP

```diff
@@ -34,4 +34,43 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
 withPipeline(type, product, component) {
     loadVaultSecrets(secrets)
     enableDbMigration()
-}
+    
+    after('test'){
+        junit 'application/build/test-results/test/**/*.xml'
```

Comment:
Did this line not work? https://github.com/hmcts/cnp-jenkins-library/blob/master/src/uk/gov/hmcts/contino/GradleBuilder.groovy#L22